docs/reference/commands/reconfigure-node.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[reconfigure-node]]
+== elasticsearch-reconfigure-node
+
+The `elasticsearch-reconfigure-node` command reconfigures an {es} node that has been installed via
+an RPM or DEB package, so that it can join an existing cluster with security features enabled.
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+-----------------------------------------------------
+bin/elasticsearch-reconfigure-node
+[--enrollment-token] [-h, --help] [-E <KeyValuePair>]
```

Comment:
If `--reconfigure` is a valid option, it needs to be added here as well.

Reply:
it is not, this is passed internally and cannot be overriden